Plugins/Object/NWScript/nwnx_object.nss

```diff
@@ -135,6 +135,8 @@ int NWNX_Object_GetHasVisualEffect(object obj, int nVFX);
 // Return TRUE if an item of baseitem type can fit in object's inventory
 int NWNX_Object_CheckFit(object obj, int baseitem);
 
+// Return damage immunity (in percent) against given damage type
+int NWNX_Object_GetDamageImmunity(object obj, int damageType);
```

Comment:
Should probably mention here that damageType takes DAMAGE_TYPE_* constants and what it actually returns.

Reply:
What's unclear about the return value description?

Reply:
I totally read over the `(in percent)`, sorry :D